WebRTCPlugin/VcmCapturer.cpp

```diff
@@ -0,0 +1,105 @@
+#include "stdafx.h"
```

Comment:
can you please explain the changes related to vcm/video capturer?

Reply:
That's weird. I thought I'd already answered this question yesterday. Or was it removed?
Anyway, the original implementation of webrtc to the same classes exist in these two sample  files. Change it however you'd like.